openlibrary/solr/update_work.py

```diff
@@ -1389,7 +1392,10 @@ def update_keys(keys,
                 output_file=None,
                 commit_way_later=False,
                 skip_id_check=False,
-                update: Literal['update', 'print', 'pprint', 'quiet'] = 'update'):
+                update: Literal['update', 'print', 'pprint', 'quiet'] = 'update',
+                # 16 threads was picked based on tests in local development.
+                # May not be optimal for all envs
+                number_threads=16):
```

Comment:
Oh, also let's expose this via the `main` function, so it's available to the CLI .

Reply:
moved